package.json

```diff
@@ -32,6 +33,7 @@
   "description": "A UI-based desktop app for BitGo Recoveries",
   "devDependencies": {
     "@playwright/test": "^1.25.2",
+    "@solana/web3.js": "^1.66.1",
```

Comment:
why do you need to install this stuff?

Reply:
Needed it for the script for building the durable nonce